compose/gettingstarted.md

```diff
@@ -86,19 +82,25 @@ itself.
 In your project directory, create a file named `Dockerfile` and paste the
 following:
 
-    FROM python:3.4-alpine
-    ADD . /code
+    FROM python:3.7-alpine
     WORKDIR /code
+    ENV FLASK_APP app.py
+    ENV FLASK_RUN_HOST 0.0.0.0
+    RUN apk add --no-cache gcc musl-dev linux-headers
+    COPY requirements.txt requirements.txt
     RUN pip install -r requirements.txt
-    CMD ["python", "app.py"]
+    COPY . .
+    CMD ["flask", "run"]
 
 This tells Docker to:
 
-* Build an image starting with the Python 3.4 image.
-* Add the current directory `.` into the path `/code` in the image.
+* Build an image starting with the Python 3.7 image.
 * Set the working directory to `/code`.
-* Install the Python dependencies.
-* Set the default command for the container to `python app.py`.
+* Set environment variables used by the `flask` command.
+* Install gcc so Python packages such as MarkupSafe and SQLAlchemy can compile speedups.
```

Comment:
nit: Since this is just a Compose get started guide, I'm not sure we need to include Python optimisations.

Reply:
Since this is one of two official Docker documents introducing devs to using Flask with Docker, I'd rather they get a reasonable experience without having to discover it later, especially since it's non-obvious that speedups are not compiled. Overall I think the focus is still on Compose.

Reply:
Do you agree, or would you like me to change something?

Reply:
My concern is that we're conflating getting started with Compose and optimising a Python app. That might overload new users with too much information.

I did LGTM the PR though as it's net better than what we have currently.

@usha-mandya is there anything else we need?

Reply:
Can't think of anything else, @chris-crone. Thanks.